tests/compute/subresources/vmi.go

```diff
@@ -181,7 +182,7 @@ var _ = compute.SIGDescribe("VirtualMachineInstance subresource", func() {
 			waitForVMIRebooted(vmi, console.LoginToFedora)
 		})
 
-		It("soft reboot vmi with ACPI feature enabled should succeed", func() {
+		It("soft reboot vmi with ACPI feature enabled should succeed", decorators.Conformance, func() {
```

Comment:
This is not optimal:
Why do we care about GA? We should explain it with `By`
Why do we sleep for 30 seconds? Flakes?